src/routes/searchPhrases.js

```diff
@@ -0,0 +1,62 @@
+import express from 'express';
+let router = express.Router();
+import knex from "../knex";
+router.get('/', (req, res) => listWrapper(req, res));
+
+//calls the database statement (createQuery) and sends the therefor needed props
+//it also waits for the result of the statement and returns the status of it
+const listWrapper = async (req, res) => {
+    const search = req.query.search;
+
+    if (search.length > 128) {
+        return res.status(400).json({success: false, error: 'Bad Request - search term is too long (max. 128 characters)'});
+    }
+
+    const city = req.query.city;
+    const language = req.query.language;
+
+    const item = await createQuery("phrase", "search_phrase", city, search, language);
+
+    const result = item;
```

Comment:
This is now redundant, you can just write:
`const result = await createQuery("phrase", "search_phrase", city, search, language);`

Hint: IntelliJ always marks you these kind of improvements with a yellow background and you can press Alt+Enter (or click the lightbulb icon) and automatically resolve this.